extensions/dash/src/util/getDashApp.ts

```diff
@@ -0,0 +1,11 @@
+import { Application, getApplications } from "@raycast/api";
+
+export default async function getDashApp(): Promise<Application> {
+  const dashApp = (await getApplications()).find((app) => app.bundleId?.indexOf("com.kapeli.dash") === 0);
```

Comment:
this could/should be cached as well, it's a hot path by the looks of it

Reply:
good point! I've updated the PR :D